Resources/ProtobufGen/protos.csv

```diff
@@ -101,6 +100,7 @@ dota2,dota_gcmessages_client_team.proto,GC\Dota\MsgGCClientTeam.cs,SteamKit2.GC.
 dota2,dota_gcmessages_client_tournament.proto,GC\Dota\MsgGCClientTournament.cs,SteamKit2.GC.Dota.Internal
 dota2,dota_gcmessages_client_watch.proto,GC\Dota\MsgGCClientWatch.cs,SteamKit2.GC.Dota.Internal
 dota2,dota_gcmessages_common.proto,GC\Dota\MsgGCCommon.cs,SteamKit2.GC.Dota.Internal
+dota2,dota_gcmessages_common_battle_report.proto,GC\Dota\MsgMCClientBattleReport.cs,SteamKit2.GC.Dota.Internal
```

Comment:
It would probably make sense to adjust the generated filename accordingly, as well